tests/unittests/hrm2_tests.py

```
@@ -0,0 +1,1171 @@
+        self.assertEqual(len(retrieved_person.smpl_params.body_pose[0][0]), 3)
+
+    @drop_datasets
+    def test_human_pose_3d_export_scene(self):
+        """Test HumanPose3D.export_scene() with Person3D objects.
+
+        This test verifies that export_scene() works with Person3D embedded
+        documents and doesn't crash with AttributeError when accessing person
+        attributes.
+        """
+        import os
+        import tempfile
```

Ruff (0.14.5)

Redefinition of unused `tempfile` from line 12

Remove definition: `tempfile`

(F811)